pkg/nodejs4

```diff
@@ -0,0 +1,30 @@
+[mirrors]
+http://nodejs.org/dist/v4.2.2/node-v4.2.2.tar.xz
+
+[vars]
+filesize=12791572
+sha512=fe6563ca641581afa4ba7fda1b25175eae474aeb196504cd82a6486bf9dfe6c753c554b31d9aa3b6453c509aa792615e897343de69a0b8bfd26759efaea2e1cb
+
+[deps.host]
+dynamic-toolchain
+python
+
+[build]
+# Better fail here, than in the middle of the build!
+if ! (gcc --version | grep -q -E '((4\.[8-9])|([5-9]\.[0-9]+))\.'); then
```

Comment:
please use `$CC` rather than gcc, because on x-compiles you test only for the host compiler